osf/models/institution.py

```diff
@@ -103,6 +103,7 @@ class Institution(DirtyFieldsMixin, Loggable, ObjectIDMixin, BaseModel, Guardian
         related_name='institutions'
     )
 
+    can_request_access = models.BooleanField(default=False, db_index=True)
```

Comment:
I don't think we need to index this field. I don't know of any use cases right now where we'd be filtering on can_request_access.